<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_definePlot</class>
 <widget class="QDialog" name="Dialog_definePlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="6" column="1">
         <widget class="QSpinBox" name="spin_leftLineThickness">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QComboBox" name="combo_leftScaleStyle"/>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QTreeWidget" name="tree_leftData">
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="indentation">
           <number>0</number>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Data</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Line color</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="btn_leftScaleColor">
          <property name="text">
           <string>Set grid color</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Unit</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Line thickness</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QComboBox" name="combo_leftUnit"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="cb_enableLeftScale">
          <property name="text">
           <string>Enable left scale</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" rowspan="2">
         <widget class="QWidget" name="widget_leftGridColor" native="true"/>
        </item>
        <item row="2" column="0" rowspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Grid style</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="5" column="1">
         <widget class="QSpinBox" name="spin_rightLineThickness">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QComboBox" name="combo_rightScaleStyle"/>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="btn_rightScaleColor">
          <property name="text">
           <string>Set grid color</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Unit</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QTreeWidget" name="tree_rightData">
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="indentation">
           <number>0</number>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <attribute name="headerStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Data</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Line color</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Line thickness</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QComboBox" name="combo_rightUnit"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="cb_enableRightScale">
          <property name="text">
           <string>Enable right scale</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QWidget" name="widget_rightGridColor" native="true"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Grid style</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Plot name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="line_name"/>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="widget_background" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn_setBackground">
       <property name="text">
        <string>Set background color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
